cqc/cqcHeader.py

```diff
@@ -272,13 +334,143 @@ def _printable(self):
         return toPrint
 
 
+class CQCTypeHeader(Header):
+    """
+        Definition of the CQC Type header. This header announces the type of the headers that will follow.
+    """
+
+    PACKAGING_FORMAT = "!BI"
+    HDR_LENGTH = struct.calcsize(PACKAGING_FORMAT)
+    
+
+    def _setVals(self, tp: CQCType=0, length: int=0) -> None:
+        """
+        Set using given values.
+        """
+        self.type = tp
+        self.length = length
+
+
+    def _pack(self) -> bytes:
+        """
+        Pack data into packet format. For defnitions see cLib/cgc.h
+        """
+        return struct.pack(self.PACKAGING_FORMAT, self.type, self.length)
+        
+
+    def _unpack(self, headerBytes) -> None:
+        """
+        Unpack packet data.
+        """
+        unpacked = struct.unpack(self.PACKAGING_FORMAT, headerBytes)
+        self.type = unpacked[0]
+        self.length = unpacked[1]
+
+
+
+    def _printable(self) -> str:
+        """
+        Produce a printable string for information purposes.
+        """
+        return "CQC Type header. Type=" + str(self.type) + " | Length=" + str(self.length)
+
+
+    def make_equivalent_CQCHeader(self, version: int, app_id: int) -> CQCHeader:
+        """
+        Produce a CQC Header that is equivalent to this CQCTypeHeader. This method does not make any modifications to self.
+        """
+        cqc_header = CQCHeader()
+        cqc_header.setVals(version, self.type, app_id, self.length)
+        return cqc_header
+        
+
+
+class CQCIFHeader(Header):
```

Comment:
Don't capitalise the 'F' in the IF - not consistent

Reply:
Changed to `CQCIfHeader`.